pkg/api/serve.go

```diff
@@ -62,7 +62,7 @@ func Serve(cfg *config.Config, catalog *catalog.Catalog, middlewareAuthenticator
 	r.Mount("/_health", httputil.ServeHealth())
 	r.Mount("/metrics", promhttp.Handler())
 	r.Mount("/_pprof/", httputil.ServePPROF("/_pprof/"))
-	r.Mount("/swagger.json", http.HandlerFunc(swaggerSpecHandler))
+	r.Mount("/openapi.json", http.HandlerFunc(swaggerSpecHandler))
```

Comment:
Is this change related?

Reply:
I used [restish](https://github.com/danielgtaylor/restish) to develop locally, I noticed they look for `/openapi.json`, also checked in the [swagger specification](https://swagger.io/specification/)
```
It is RECOMMENDED that the root OpenAPI document be named: openapi.json or openapi.yaml.
```
So, for your question, it's not exactly related, but I believe it's good to add it. 
I can remove it if you prefer

Reply:
I don't think it's not a good idea, but the question is should it be replaced or added in addition due to backwards compatibility (@nopcoder )

Reply:
It is not part of the api/v1, we can document it in the release notes - but I don't see an issue fixing this link to be more aligned with openapi recommendations.